mysql/mysqldump.go

```diff
@@ -466,7 +468,8 @@ func handleSpatialDatatype(conv *internal.Conv, chunk string, l [][]byte) ([]ast
 		return nil, true
 	}
 	for _, spatial := range MysqlSpatialDataTypes {
-		chunk = strings.ReplaceAll(chunk, " "+spatial, " text")
+		spatialRegexp := regexp.MustCompile("(?i)" + " " + spatial)
```

Comment:
Can we move this to a global var? E.g., 

```go
var MysqlSpatialDataTypes = []string{"geometrycollection", "multipoint", "multilinestring", "multipolygon", "point", "linestring", "polygon", "geometry"}

var spatialRegexps = func() []*regexp.Regexp {
	l := make([]*regexp.Regexp, len(MysqlSpatialDataTypes))
	for i, spatial := range MysqlSpatialDataTypes {
		l[i] = regexp.MustCompile("(?i)" + " " + spatial)
	}
	return l
}()
```

This would bring `regexp.MustCompile` to the build time instead of runtime and can early detect errors if exist.

Reply:
Sure, I have moved it to a global variable as per your suggestion.